### ISSUE TITLE
Mingw32: fltk-config regression with staticflags for local png/zlib

### ISSUE THREAD

**Issue by wcout** [fltk/fltk#717]
**Describe the bug**
fltk-config does not correctly output the **static** local png/zlib flags.

**To Reproduce**
````
./configure --host=i686-w64-mingw32 --enable-cygwin --enable-localjpeg --enable-localpng --enable-localzlib
````

````
./fltk-config --use-images --ldstaticflags
-mwindows ./lib/libfltk_images.a /usr/local/lib/libfltk_png.a /usr/local/lib/libfltk_z.a /usr/local/lib/libfltk_jpeg.a ./lib/libfltk.a -lgdiplus -lgdiplus -lole32 -luuid -lcomctl32 -lws2_32
````

**Expected behavior**
````
./fltk-config --use-images --ldstaticflags
-mwindows ./lib/libfltk_images.a ./lib/libfltk_png.a ./lib/libfltk_z.a ./lib/libfltk_jpeg.a ./libfltk.a -lgdiplus -lgdiplus -lole32 -luuid -lcomctl32 -lws2_32
````

**FLTK Version**
 1.4.0, Git, 3835b72e2

**Operating System / Platform:**
Linux: Ubuntu 20.04

**Linux/Unix Runtime, if applicable:**
 - mingw32

**Additional context**
I updated from version git commit cce40848c to this version. The older version does not have this bug.

**Comment by Albrecht-S** [comment]
Thanks for the report and sorry for the inconveniencies. This was indeed a regression of the changes done for issues #647 and #656.

This bug should be fixed with commit f9669b80b95fea8d7f6426340c5616ed46e90daa. Please confirm and close the issue if it's OK. Thanks.

**Comment by wcout** [comment]
Yes, that fixed it. Thanks.
